packages/dev/core/src/Events/deviceInputEvents.ts

```diff
@@ -156,23 +156,27 @@ export interface IMouseEvent extends IUIEvent {
 
     /**
      * Delta of movement on X axis
+     * @deprecated

```

Comment:
nit: Maybe good to put a message.
```suggestion
     * @deprecated Use `movementX` instead
```

Reply:
Added